### PR TITLE
LMB-428: Redesign details page of mandataris

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

Following the figma design for the details of a mandataris.

History table didn't change except for the subrows as we cannot do this atm.

## How to test

Click throught the page and everything should work the same. 

## Attachments 

image

### CODE REVIEW

File: app/templates/mandatarissen/persoon/mandataris.hbs [hunk 1]
```diff
@@ -4,19 +4,40 @@
...
+      >Wijzig huidig mandaat</AuButton>
+      <AuButton
+        @skin="secondary"
+        {{on "click" (fn (mut this.isCorrecting) true)}}
+      >Corrigeer fouten</AuButton>
+    </div>
+    <AuAlert @skin="info" @icon="info-circle">
+      Om het mandaat te updaten met nieuwe informatie, zoals een
+      fractiewijziging, kies voor
+      <span class="au-u-medium">Wijzig huidig mandaat</span>. Dit zal een nieuwe
```
Comment (Rahien) [inline app/templates/mandatarissen/persoon/mandataris.hbs:32]:
```suggestion
      <span class="au-u-medium">Wijzig huidig mandaat</span>. Dit zal een nieuw
```

File: app/templates/mandatarissen/persoon/mandataris.hbs [hunk 2]
```diff
@@ -4,19 +4,40 @@
...
+        class="au-u-margin-right-small"
+        @disabled={{not this.model.mandataris.isActive}}
+        {{on "click" (fn (mut this.isChanging) true)}}
+      >Wijzig huidig mandaat</AuButton>
+      <AuButton
+        @skin="secondary"
+        {{on "click" (fn (mut this.isCorrecting) true)}}
+      >Corrigeer fouten</AuButton>
+    </div>
+    <AuAlert @skin="info" @icon="info-circle">
```
Comment (Rahien) [inline app/templates/mandatarissen/persoon/mandataris.hbs:29]: the icon in the design is dark, this one is light, can we change that?

File: app/components/mandatarissen/mandataris-card.hbs [hunk 3]
```diff
@@ -1,116 +1,130 @@
...
+      <div class="au-c-description-value au-u-margin-top-tiny">
+        {{#if @mandataris.einde}}
+          {{moment-format @mandataris.einde "DD-MM-YYYY"}}
+        {{else}}
+          heden
+        {{/if}}
       </div>
+    </div>
+    <div class="au-o-grid__item au-u-5-6">
+      <div class="au-c-description-label"><div>Beleidsdomein(en)</div></div>
```
Comment (Rahien) [inline app/components/mandatarissen/mandataris-card.hbs:75]: can we only show this if the mandataris classificatie is schepen or burgemeester or aangestelde burgemeester or districtsburgemeester?

File: app/components/mandatarissen/mandataris-card.hbs [hunk 4]
```diff
@@ -1,116 +1,130 @@
...
+        </p> }}
       </div>
-      <div class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
-        <div class="au-c-description-label">Startdatum</div>
-        <div class="au-c-description-value">
-          {{moment-format @mandataris.start "DD-MM-YYYY"}}
-        </div>
+    </div>
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Rangorde</div>
```
Comment (Rahien) [inline app/components/mandatarissen/mandataris-card.hbs:47]: can we only show this for schepenen?

File: app/components/mandatarissen/mandataris-card.hbs [hunk 5]
```diff
@@ -1,116 +1,130 @@
...
+          @iconAlignment="left"
+          @hideText={{false}}
+        >
+          {{if
+            @mandataris.publicationStatus
+            @mandataris.publicationStatus.label
+            "Bekrachtigd"
+          }}
+        </AuPill>
+        {{! <AuLinkExternal href="" class="au-u-margin-left-tiny">Bekijk bewijs</AuLinkExternal> }}
```
Comment (Rahien) [inline app/components/mandatarissen/mandataris-card.hbs:34]: why comment this? i'd rather we leave it in and put a todo on the href if that's what we're waiting for. Just add a bug ticket for it, it's super small and this way it's tracked

File: app/components/mandatarissen/mandataris-card.hbs [hunk 6]
```diff
@@ -1,116 +1,130 @@
...
+      >{{this.status}}</div>
+    </div>
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Publicatie status</div>
+      <div class="au-c-description-value au-u-margin-top-tiny">
+        <AuPill
+          @skin="link"
+          @draft={{false}}
+          @size="regular"
+          @icon="info-circle"
```
Comment (Rahien) [inline app/components/mandatarissen/mandataris-card.hbs:24]: the design doesn't show an icon

File: app/components/mandatarissen/mandataris-card.hbs [hunk 7]
```diff
@@ -1,116 +1,130 @@
...
+      <div class="au-c-description-label">Status</div>
+      <div
+        class="au-c-description-value au-u-margin-top-tiny"
+      >{{this.status}}</div>
+    </div>
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Publicatie status</div>
+      <div class="au-c-description-value au-u-margin-top-tiny">
+        <AuPill
+          @skin="link"
```
Comment (Rahien) [inline app/components/mandatarissen/mandataris-card.hbs:21]: different skin based on the code. I think i build something similar in my pr, you can extract that as a component, wdyt?

File: app/components/mandatarissen/mandataris-card.hbs [hunk 8]
```diff
@@ -1,116 +1,130 @@
...
-        <div class="au-c-description-value"><Mandaat::FoldedFracties
-            @persoon={{this.persoon}}
-            @bestuursperiode={{@bestuursperiode}}
-          /></div>
+<div class="au-o-box au-u-background-gray-100">
+  <div class="au-o-grid au-o-grid--tiny">
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Status</div>
+      <div
+        class="au-c-description-value au-u-margin-top-tiny"
```
Comment (Rahien) [inline app/components/mandatarissen/mandataris-card.hbs:14]: design has this as a pill too

File: app/components/mandatarissen/mandataris-card.hbs (hunk 3 above)
Comment (Rahien) [reply app/components/mandatarissen/mandataris-card.hbs:75]: nvm about this one, should be tackled by another pr

**Review by Rahien — CHANGES_REQUESTED** [review]

File: app/templates/mandatarissen/persoon/mandataris.hbs (hunk 2 above)
Comment (JonasVanHoof) [reply app/templates/mandatarissen/persoon/mandataris.hbs:29]: Didn't see an option for this... And when trying with custom css its only p[ossible as a square ;p 
image

File: app/components/mandatarissen/mandataris-card.hbs (hunk 7 above)
Comment (JonasVanHoof) [reply app/components/mandatarissen/mandataris-card.hbs:21]: Create a component for this and also use it now in the persoon mandaten table of yours

File: app/components/mandatarissen/mandataris-card.hbs (hunk 8 above)
Comment (JonasVanHoof) [reply app/components/mandatarissen/mandataris-card.hbs]: Updated it. Doe sit need extra logic for colors?

File: app/components/mandatarissen/mandataris-card.hbs (hunk 5 above)
Comment (JonasVanHoof) [reply app/components/mandatarissen/mandataris-card.hbs]: Add ticket here LMB-481, LMB-482

File: app/components/mandatarissen/mandataris-card.hbs (hunk 8 above)
Comment (JonasVanHoof) [reply app/components/mandatarissen/mandataris-card.hbs]: Effectief status will be green the rest is gray

File: app/templates/mandatarissen/persoon/mandataris.hbs (hunk 2 above)
Comment (JonasVanHoof) [reply app/templates/mandatarissen/persoon/mandataris.hbs]: image

**Review by Rahien — APPROVED** [review]

**Pull request merged**